### PR TITLE
Players can Ctrl+Shift click APCs to toggle night shift lighting

### LINKED ISSUE #39897: Night shift lighting toggle cooldown doesn't work
https://github.com/tgstation/tgstation/blob/30dd028e5027d8b5a60d77b1596cc0b963e131f5/code/modules/power/apc.dm#L914-L917

The 100 tick cooldown in between allowed toggles doesn't work; players can freely spam it.

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Players can now ctrl+shift click APCs to toggle night shift lighting.
fix: The 10 second anti spam cooldown of night shift lighting now works properly and has been increased to 60 seconds.
/:cl:

Closes: #39897

I've heard from plenty of players that toggling night lighting is frustrating, as they need to have access/hack the APC and then go through an interface to toggle what's essentially a light switch.
This PR lets them do it by ctrl+shift-clicking APCs - no access check, since it just toggles cosmetic lighting.

### CODE REVIEW

**Comment by adamkad1** [comment]
yisssssss

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -264,6 +264,9 @@
 
 	if(integration_cog && is_servant_of_ratvar(user))
 		to_chat(user, "<span class='brass'>There is an integration cog installed!</span>")
+	
+	to_chat(user, "<span class='notice'>Alt+Click [src] to toggle the cover lock.</span>")
+	to_chat(user, "<span class='notice'>Ctrl+Click [src] to toggle night shift lighting.</span>")
```
Comment (nicbn) [inline code/modules/power/apc.dm]: Ctrl+Shift

Comment (81Denton) [reply code/modules/power/apc.dm]: oh

**Comment by GuppyLaxx** [comment]
Just remove this garbage feature already

**Comment by zxaber** [comment]
And queue the assistants fighting over whether the main hallway is in night mode.
Or even better, queue the assistants sitting in maint on Box and making rooms flicker.

Still, I appreciate this change. I like the night-lighting in the main hallway, but I prefer my lab lit up.

**Comment by 81Denton** [comment]
Increasing the cooldown to something like a minute is also fine by me

**Comment by PKPenguin321** [comment]
I was thinking turning it off would require no access, but turning it on would, to prevent what @zxaber said. But I don't mind too much

**Comment by zxaber** [comment]
Presumably, at least some players would like it on. Otherwise, why have it at all?

I imagine the best solution, from the user's standpoint, would be to have this setting linked to access of the area (only people with medical access can adjust medical area night-mode). But that would probably be a lot of extra code (not to mention map edits) for what amounts to a simple fluff feature.

**Comment by optimumtact** [comment]
I like night-lighting so I"m opposed to this change :-1:

**Comment by 81Denton** [comment]
It would require someone to either code APCs to check for the current area and assign access according to it, or manually assign access to every single one.
Option A) will break for non-standard areas and option B) is a nightmare to implement and maintain.

**Comment by 81Denton** [comment]
@optimumtact I like night lighting as well. I've heard from a bunch of people that they have a hard time seeing when it's active, so imo it doesn't hurt if they have an easier time disabling it.
At the least, I think that this cosmetic feature doesn't need to be protected by the APC cover lock.

**Comment by PKPenguin321** [comment]
@optimumtact this makes it easier to enable night lighting too

**Comment by Cyberboss** [comment]
Not in favor of removing the access restriction

**Pull request closed without merging**